### PR TITLE
Adding ~'* keyword

### LINKED ISSUE #10: Add `~'*` to the keyword list
Samarium 0.4.0 will introduce static methods represented by the `~'*` token. Currently, they're highlighted separately as operators and keywords. It should instead be treated as its own keyword.

### PR DESCRIPTION
**Pull request by sujal-goswami**
This closes #10

### CODE REVIEW

File: syntaxes/samarium.tmLanguage.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
                 },
                 {
                     "name": "keyword.control.samarium",
-                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%|<=)"
+                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%|<=|~'*)"
```
Comment (trag1c) [inline syntaxes/samarium.tmLanguage.json]: Keep in mind that `*` is a quantifier, so it needs to be escaped
```suggestion
                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%|<=|~'\\*)"
```

Comment (sujal-goswami) [reply syntaxes/samarium.tmLanguage.json]: thanks, I have learned so much from these two PR's .

Comment (trag1c) [reply syntaxes/samarium.tmLanguage.json]: Glad to hear that!

**Review by trag1c — APPROVED** [review]

**Comment by sujal-goswami** [comment]
hey can you have any idea why it shows this
image

**Comment by zunda-arrow** [comment]
it takes 15 minutes to update

**Pull request merged**